app/app.go

```diff
@@ -213,7 +213,7 @@ func init() {
 		panic(err)
 	}
 
-	DefaultNodeHome = filepath.Join(userHomeDir, "."+Name)
+	DefaultNodeHome = filepath.Join(userHomeDir, "."+Name+"d")
```

Comment:
Why are we changing the node home to cardchain`d`?

Reply:
Thats currently the standard in most cosmos projects i checked